config/environments/production.rb

```diff
@@ -53,7 +53,7 @@
   # Prevent host header injection
   config.action_controller.default_url_options = { host: ENV['SITE_URL'] }
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  config.action_controller.asset_host = ENV['SITE_URL']
```

Comment:
is this a two-part roll out where eventually we can drop the `|| ENV['SITE_URL']`, or do we always want the fall back for local dev?

Reply:
I think keep the fallback for the time being - this is pretty specifically for multitenant which otherwise tries to grab assets from weird places. Maybe we'll revisit after we've consolidated.